@AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageASEDraw.sqf

```diff
@@ -45,4 +46,83 @@ _msn_equip_British = _heli animationPhase "msn_equip_british";
 _heli setUserMfdValue  [MFD_INDEX_OFFSET(MFD_IND_ASE_BRITISH), _msn_equip_British];
 
 _msn_equip_american = _heli animationPhase "msn_equip_American";
-_heli setUserMfdValue  [MFD_INDEX_OFFSET(MFD_IND_ASE_AMERICAN), _msn_equip_american];
\ No newline at end of file
+_heli setUserMfdValue  [MFD_INDEX_OFFSET(MFD_IND_ASE_AMERICAN), _msn_equip_american];
+
+//ASE Points
+private _pointsArray = [];
+private _aseObjects  = _heli getVariable "fza_ah64_ase_rlwrObjects";
+private _radius      = 0.27;
+{
+    _x params ["_state", "_bearing", "_classification"];
+    private _ident      = "";
+    private _unitStatus = "";
+    private _unitType   = "";
+    //Unit type
+    switch (_classification) do {
+        case "sa2": {
+            _unitType = "SA2";
+        };
+        case "sa3": {
+            _unitType = "SA3";
+        };
+        case "sa8": {
+            _unitType = "SA8";
+        };
+        case "sa10": {
+            _unitType = "SA10";
+        };
+        case "sa15": {
+            _unitType = "SA15";
+        };
+        case "sa17": {
+            _unitType = "SA17";
+        };
+        case "sa19": {
+            _unitType = "SA19";
+        };
+        case "sa20": {
+            _unitType = "SA20";
+        };
+        case "sa21": {
+            _unitType = "SA21";
+        };
+        case "gun": {
+            _unitType = "GU";
+        };
+        case "hawk": {
+            _unitType = "HK";
+        };
+        case "naval": {
+            _unitType = "NV";
+        };
+        case "2S6": {
+            _unitType = "2S6";
+        };
+        case "radar": {
+            _unitType = "SR";
+        };
+        case "zsu": {
+            _unitType = "ZU";
+        };
+    };
+    //Unit status
+    switch (_state) do {
+        case ASE_SRH: {
+            _unitStatus = "SRH";
+        };
+        case ASE_ACQ: {
+            _unitStatus = "ACQ_TRK";
+        };
+        case ASE_TRK: {
+            _unitStatus = "ACQ_TRK";
+        };
+        case ASE_LNC: {
+            _unitStatus = "LNC";
+        };
+    };
+
+    _ident           = (["RLWR", _unitType,_unitStatus]) joinString "_";
```

Comment:
Could you unity the creation of this ident between TSD and ASE into a single function, as it looks identical?